addons/test_mail/tests/test_mail_mail.py

```diff
@@ -56,6 +56,33 @@ def test_mail_mail_return_path(self):
             mail.send()
         self.assertEqual(self._mails[0]['headers']['Return-Path'], '%s@%s' % (self.alias_bounce, self.alias_domain))
 
+    def test_mail_mail_fields_accessible(self):
```

Comment:
Linting: order test methods alphabetically in a given class, so that it matches execution order (easier when debugging tests). Could be ``test_mail_mail_fields_access`` by the way, to be shorter :)